pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = [
+    "cmake>=3.19",
+    "numpy>=1.21.2",
+    "scikit-build-core >=0.4.3",
+    "ninja; platform_system!='Windows'"
+]
+
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "finufft"
+version = "0.0.1"
+description = "Python bindings for the FINUFFT C++ library"
+readme = "README.md"
+requires-python = ">=3.8"
+
+[tool.scikit-build]
+# Protect the configuration against future changes in scikit-build-core
+minimum-version = "0.4"
+cmake.targets = ["finufft"]
+cmake.args = ['-DFINUFFT_BUILD_PYTHON=ON', '-DCMAKE_BUILD_TYPE=Release'] # not sure if this is necessary
+wheel.packages = ["python/finufft/finufft"]
+wheel.exclude = ["**.pyx"]
```

Comment:
Is this needed? Do we have any `pyx` files?

Reply:
Are they not generated by pip?

Reply:
I think pyx files are used to contain Cython source code, which we shouldn't have here.

Reply:
Removed.